eng/linkerTests/project.csproj.template

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Please don't hardcode the tfm here, instead use `{TargetFramework}` and replace it with `NetCoreAppCurrent`.